### DISCUSSION TITLE
Program Search

### DISCUSSION THREAD

**Discussion by Scabad** [valinet/ExplorerPatcher#2867] [General]
Hi All

I have just upgraded to Windows 11 Pro, having built a new desktop system.
I have installed ExplorerPatcher, as I really couldn't cope with the Windows 11 interface. 
Below are the details of my OS and EP version Number

![EP Version](https://github.com/valinet/ExplorerPatcher/assets/160549315/04193a57-fd92-400a-b03a-c1d061976202)

Initially when I set up EP it seemed to be working exactly as I wanted it to, as in I clicked the start button, bottom left, and the program list appeared, and when I started to search for an option e.g. Settings, the search bar appeared correctly at the top of the box on the left.

![Step 1](https://github.com/valinet/ExplorerPatcher/assets/160549315/1112c42a-a976-46e7-93b7-39976cdb5c9f)

However, after a restart, when I start to type the box moves into the centre of the screen in a windows 11 format.

![Step 2](https://github.com/valinet/ExplorerPatcher/assets/160549315/ee4a27c8-20d7-46ed-9fda-0cafbfa63933)

I have tried quite a few options but with no luck, I have searched here on the forum, but cannot find anything regarding this to point me in the right direction.

Any help would be very much appreciated.
Thanks in advance
Scabad

**Comment by pyrates999** [comment]
After closing the start menu and clicking on it again, does it then show up on the left side?

**Comment by Scabad** [comment]
@pyrates999 yes it goes back to the left it's only when I type to search e.g. settings, power plan, etc. that it moves to the centre

**Reply by pyrates999** [reply]
Another issue brought this up and I remember it was closed as won't fix.  Sorry.  The maintainer would know which issue it was.

**Reply by Scabad** [reply]
As long as I know I'm not missing something I don't mind, I thought it was something I was doing.
Thanks for your input :-)
